.gitignore

```diff
@@ -1,10 +1,6 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
-composer.lock
-vendor
+/build
+/Resources/doc/_build/
+/vendor
 .php_cs.cache
+composer.lock
+phpunit.xml
```

Comment:
Please configure your IDE to add a newline at end of file, it should make the red pictogram disappear